testData/preferences/phil.json

```diff
@@ -0,0 +1,13 @@
+{
+    "flat": {
+        "contexts": {
+            "gpii-default": {
+                "name": "Default preferences",
+                "preferences": {
+                    "http://registry.gpii.net/common/slowKeysEnable": true,
```

Comment:
This is perhaps not the correct pull request for this, but in my windows 10 box, when logging in with phil, and look at the settings, filter keys is not really turned on, what IS turned on is the following options:
* Turn on Filter Keys when right SHIFT is pressed for 8 seconds
* Beep when keys are pressed or accepted
* Display the Filter Keys icon on the task bar

Whereas the following are not enabled:
* Ignore or slow down brief or repeated keystrokes and adjust keyboard repeat rates
* Enable Slow Keys

I can still merge this (since these are just NP sets) and then create a JIRA for the issue - but I wanted to check with you that this indeed is NOT and issue with the NP set, but rather with the solution entry (or my windows box)